packages/dev/s2-docs/pages/s2/Skeleton.mdx

```diff
@@ -12,12 +12,12 @@ export const description = 'Wraps around content to render it as a placeholder.'
 
 ```tsx render docs={docs.exports.Skeleton} links={docs.links} props={['isLoading']} initialProps={{isLoading: true}} type="s2" wide
 "use client";
-import {Skeleton, Image, Heading, Text} from '@react-spectrum/s2'
+import {Skeleton, Image, Text} from '@react-spectrum/s2'
 import {style} from '@react-spectrum/s2/style' with {type: 'macro'};
 import preview from 'url:./assets/preview.png';
 import Select from '@react-spectrum/s2/icons/Select';
 
-<Skeleton/* PROPS */>
+<Skeleton isLoading={true}/* PROPS */>
```

Comment:
This causes isLoading to appear twice in the skeleton code sample:

<img width="833" height="472" alt="Image" src="https://github.com/user-attachments/assets/86c74da8-d868-4c0f-a2b1-c857a3c052d5" />